.gitignore

```diff
@@ -4,3 +4,5 @@ __pycache__/
 *.pyo
 .vscode/
 .pytest_cache/
+.vscode
```

Comment:
`.vscode/` is already there. Any specific reasons for adding this once again?

Reply:
Just a small oversight, corrected.